packages/suite/src/views/wallet/coinmarket/common/CoinmarketTransactions/CoinmarketTransaction/CoinmarketTransactionContainer.tsx

```diff
@@ -16,10 +16,10 @@ export const CoinmarketTransactionContainer = ({
     const isMobile = useMediaQuery(`(max-width: ${variables.SCREEN_SIZE.SM})`);
 
     return (
-        <Card fillType="none">
+        <Card fillType="none" margin={{ bottom: spacings.lg }}>
             <Row flexWrap={isBelowDesktop ? 'wrap' : undefined}>
                 <Column
-                    alignItems="flex-start"
+                    alignItems="stretch"
```

Comment:
You can delete `alignItems` here. Yesterday we switched `Column` default value to `normal` which is basically the same like `stretch`.